compiler/src/dotty/tools/repl/ReplDriver.scala

```diff
@@ -307,6 +308,7 @@ class ReplDriver(settings: Array[String],
           .membersBasedOnFlags(required = Method, excluded = Accessor | ParamAccessor | Synthetic | Private)
```

Comment:
I would have expected the exclusion of `Synthetic` to be enough to avoid default getters, so I think the real issue is that they're not always marked Synthetic, it looks like currently they're only Synthetic if the method they correspond to is also Synthetic, but it should be unconditional: https://github.com/lampepfl/dotty/blob/af3703bc4c489506afb2461e1f7cea72984ea1a0/compiler/src/dotty/tools/dotc/ast/Desugar.scala#L302